### PR TITLE
MBS-9909: Change reports to use React (12 - places)

### PR DESCRIPTION
**Pull request by reosarevok**
To be reviewed and merged after https://github.com/metabrainz/musicbrainz-server/pull/930

### CODE REVIEW

File: root/report/DeprecatedRelationshipPlaces.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  items,
+  pager,
+}: ReportDataT<ReportPlaceRelationshipT>) => (
+  <Layout fullWidth title={l('Places with deprecated relationships')}>
+    <h1>{l('Places with deprecated relationships')}</h1>
+
+    <ul>
+      <li>
+        {l(`This report lists places which have relationships using
+            deprecated and grouping-only relationship types.`)}
```
Comment (yvanzo) [inline root/report/DeprecatedRelationshipPlaces.js]: Added period.

File: root/report/PlacesWithoutCoordinates.js [hunk 2]
```diff
@@ -66,7 +68,7 @@ const PlacesWithoutCoordinates = ({
                 <td>{item.place.address}</td>
                 <td>
                   {item.place.area
-                    ? <EntityLink entity={item.place.area} />
+                    ? <DescriptiveLink entity={item.place.area} />
```
Comment (yvanzo) [inline root/report/PlacesWithoutCoordinates.js]: Is https://github.com/metabrainz/musicbrainz-server/pull/931/commits/a16b759572eaae9b07cd3cdbbdf016f99f487416 supposed to show area’s parents or something? I cannot see any difference.

Comment (reosarevok) [reply root/report/PlacesWithoutCoordinates.js]: Yes. It used to work, but I'll re-check in case it now doesn't.

Comment (yvanzo) [reply root/report/PlacesWithoutCoordinates.js]: Alright, areas are just not linked to countries in sample data.

**Pull request merged**